### PR TITLE
Updating docs for changes with Operator 0.19.5

### LINKED ISSUE #527: [Release] Updates in Operator 0.19.5
New functionality has been added in https://github.com/minio/console/releases/tag/v0.19.5 of the Operator.

- [x] Update the appropriate MinIO Operator screen with information on registering tenants by API key. [PR #2222](https://github.com/minio/console/pull/2222/)
- [ ] ~~Operator can now start in DirectPV mode [PR #2203](https://github.com/minio/console/pull/2203)~~
- [x] New fields to support FSGroupChangePolicy when creating a tenant[PR #2233](https://github.com/minio/console/pull/2233)
- [x] When retrieving an API key from Subnet, the tenant can now auto register [PR #2217](https://github.com/minio/console/pull/2217)
- [x] Update screenshots for identity provider reflecting logos [PR #2210](https://github.com/minio/console/pull/2210)
- [x] When creating buckets, updated what is available to use based on permissions [PR #2226](https://github.com/minio/console/pull/2226)

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #527

- Adds information about the new Register tab in the Operator
- Updates screenshots to show the Register tab
- Adds note about permissions when managing buckets in a tenant
- Added new FSGroupChangePolicy field to security context when creating tenants

### CODE REVIEW

File: source/operations/install-deploy-manage/minio-operator-console.rst [hunk 1]
```diff
@@ -63,22 +63,41 @@ The Operator Console automatically detects any MinIO Tenants
...
-   :alt: MinIO Operator Tenant Console
+Tenant Registration
+-------------------
 
-Select :guilabel:`VIEW` to view the Tenant details and configurations. 
-You can modify, expand, upgrade, and delete the Tenant from this view.
+.. versionchanged:: 0.19.5
 
-.. image:: /images/k8s/operator-tenant-view.png
+   You can register the the Operator's tenants with an existing |SUBNET| subscription within the Operator Console.
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/minio-operator-console.rst:74]:
```suggestion
   You can register your MinIO tenants with your |SUBNET| account using the Operator Console.
```

**Review by ravindk89 — APPROVED** [review]
LGTM mod nits.

File: source/operations/install-deploy-manage/minio-operator-console.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/minio-operator-console.rst:74]: 👍

File: source/operations/install-deploy-manage/minio-operator-console.rst [hunk 2]
```diff
@@ -63,22 +63,41 @@ The Operator Console automatically detects any MinIO Tenants
...
    :class: no-scaled-link
-   :alt: MinIO Operator Tenant View
+   :alt: MinIO Operator Console Register Screen
+
+#. Select the :guilabel:`Register` tab
+#. Enter the :guilabel:`API Key` if you already have it
+   
+   or
+   
+   Select :guilabel:`Get from SUBNET` to obtain an API key
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/operations/install-deploy-manage/minio-operator-console.rst:87]: Minor tweaks, but merged.

File: source/operations/install-deploy-manage/minio-operator-console.rst [hunk 3]
```diff
@@ -63,22 +63,41 @@ The Operator Console automatically detects any MinIO Tenants
...
+#. Select :guilabel:`Register`
+
+Review Your MinIO License
+-------------------------
+
+View the license you are using for your MinIO deployments.
+MinIO is provided as Open Source Software under the terms of the AGPLv3 license.
+Subscribers to |SUBNET| use MinIO under a commercial license.
+
+To review which license you are using and the features available through different license options, select the :guilabel:`License` tab.
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/operations/install-deploy-manage/minio-operator-console.rst:101]: Done

**Pull request merged**